packages/connect-wallet-modal/src/connectButtons/connectLedger.tsx

```diff
@@ -16,6 +16,7 @@ const ConnectLedger: FC<ConnectWalletProps> = (props) => {
       setRequirements(true, {
         icon: <WalletIcon />,
         title: "Ledger couldn't connect",
+        text: "Your browser doesn't support direct connection with Ledger. Please, try another browser.",
```

Comment:
Maybe add any recommendations, what browser to try? Maybe list of browsers supported by ledger?

Reply:
Any browser with WebHID API will work. I don't want to support such browsers list here. It is a rare case anyway.